sections/introduction.tex

```diff
@@ -0,0 +1,31 @@
+\section{Introduction}\label{sec:introduction}
+Datasets consist of entries where each entry contains a certain amount of meta-data fields which describes the data.
+Datasets are sometimes not used to their full potential, regarding the amount of information available within some datasets.
+There exist many feature engineering method, which capture different aspects of a dataset, and these might aid in solving a specific task.
+Meta-data is describes field within a data entry which and is usually used with supervised learning as labels for the algorithm used.
+These meta-data fields within a data entry can be used for many purposes eg. to predict new samples within a certain scenario.
+
+Within the field of topic modelling, \Gls{lda} proven successful in finding underlying topics within a corpus of text documents.
+These topics are based on the document that resides in the corpus and words within these documents, but other extensions of \gls{lda} have included various other information such as authors, into the model, to generate better topics.
+One of the well known extensions is the Author-Topic model by \citet{author_topic} which combines the \gls{lda} model which models the relationship between authors and the documents, which they have written.
+The reason for modelling this relationship is to figure out which subjects certain authors write about and how much each author use specific words. 
+They also show that the Author-Topic combination yields better and more coherent topics, which begs the question whether any other metadata can be applied in a similar fashion.
+What if the underlying topics, within a corpus could found with the help of features already present in the data.
+
+Nordjyske is a Danish news agency who maintain multiple newspapers throughout north Jutland, a region in Denmark.
+They store their news articles within a database where they save multiple meta-data about each article.
+One of the metadata fields is the Category field, which both describes where the article is supposed to be located(within a newspaper) and also which subject the article is about. 
+These meta-data are very interesting in that they detail the data in certain ways which might be useful in some way.
```

Comment:
intetsigende sætning. slutningen med 'in certain ways which might be useful in some way' er for vag